README.md

```diff
@@ -117,6 +117,8 @@ Buttercup for Desktop supports the following languages:
  * Polish
  * Finnish
  * Catalan
+ * Catalan
```

Comment:
This is duplicated

Reply:
sry,I have been busy  lately, I just saw the news，I will fix it right away.